spec/forms/work_form_spec.rb

```diff
@@ -46,31 +50,23 @@
 
     it 'filters out name values for the wrong type' do
       form.validate(contributors: contributors)
-      expect(form.contributors.size).to eq 2
+      expect(form.contributors.size).to eq 3
       expect(form.contributors.first.full_name).to be_nil
-      expect(form.contributors.last.first_name).to be_nil
-      expect(form.contributors.last.last_name).to be_nil
+      expect(form.contributors.first.first_name).to be_nil
+      expect(form.contributors.first.last_name).to be_nil
+      expect(form.contributors.second.full_name).to be_nil
+      expect(form.contributors.third.first_name).to be_nil
+      expect(form.contributors.third.last_name).to be_nil
     end
   end
 
   describe 'contributors validation' do
     let(:contributor_error) { { contributors: ['Please add at least one contributor.'] } }
+    let(:work) { build(:valid_work) }
 
-    context 'with no contributors' do
-      it 'does not validate' do
```

Comment:
No, the contributor prepopulator prevents this from ever being the case, so remove the test as this is not a behavior in deployed instances.

Reply:
Prepopulate only happens on show, not on update, right?

Reply:
@jcoyne I mean, this test fails now due to the prepopulator change. We prepopulate when [rendering the new action](https://github.com/sul-dlss/happy-heron/blob/78f978ffcfbbbe50b505c8da65b3bbaa48b3050b/app/controllers/works_controller.rb#L19) and when [rendering the edit action](https://github.com/sul-dlss/happy-heron/blob/78f978ffcfbbbe50b505c8da65b3bbaa48b3050b/app/controllers/works_controller.rb#L41).

Reply:
@jcoyne see the change I made above on line 12.